lib/src/tagger.dart

```diff
@@ -425,6 +425,11 @@ class _FlutterTaggerState extends State<FlutterTagger> {
         selectionOffset++;
       }
 
+      // if (text.length - 1 == position || text[position + 1] != ' ') {
```

Comment:
Any reason for this change?

Reply:
I was trying to confirm something else with regards to space in the tag name. I will leave that for another issue.

Reply:
Reverted the change.